Arduino/motor_direction_test/motor_direction_test.ino

```diff
@@ -0,0 +1,36 @@
+// Test which direction the motor spins based on GPIO High or Low
+
+/*
+Results: 
+
+High ==> CW
+LOW ==> CCW
+
+*/
+
+//Initializing Pin
+const int PWM_PIN = 11;
+const int DIR_PIN = 10;
+bool flag = HIGH;
+
+void setup() {
+  Serial.begin(9600);
+  pinMode(PWM_PIN, OUTPUT);
+  pinMode(DIR_PIN, OUTPUT); //direction control PIN 10 with direction wire 
+
+}
+
+void loop() {
+
+  // Alternate direction
+  flag = flag ^ HIGH;
```

Comment:
Can you explain why we XOR this?

Reply:
Different way of doing `flag = !flag` - this was just to flip between 0 & 1 to alternative directions.